ament_cmake_google_benchmark/ament_cmake_google_benchmark-extras.cmake

```diff
@@ -16,6 +16,9 @@ macro(_ament_cmake_google_benchmark_find_benchmark)
   if(NOT DEFINED _AMENT_CMAKE_GOOGLE_BENCHMARK_FIND_BENCHMARK)
     set(_AMENT_CMAKE_GOOGLE_BENCHMARK_FIND_BENCHMARK TRUE)
 
+    option(AMENT_RUN_PERFORMANCE_TESTS
+      "When disabled, performance tests are unconditionally skipped" OFF)
```

Comment:
Imo "When disabled ..." implies that the user has to do something to achieve this but it is the default. Maybe:

> Enable performance tests instead of unconditionally skipping them

Reply:
ac1f309